python_modules/dagster/dagster/_core/execution/plan/compute_generator.py

```diff
@@ -252,7 +266,7 @@ def validate_and_coerce_op_result_to_iterator(
             "returning an AssetMaterialization "
             "or an ExpectationResult from "
             f"{context.op_def.node_type_str} you must yield them "
-            "directly, or log them using the OpExecutionContext.log_event method to avoid "
+            "directly, or log them using the ExecutionContexts.log_event method to avoid "
```

Comment:
should this be "execution context log_event method" since the specific context is unknown? this currently reads weird